From ab4c282729847338e97d236342d21fabd7cfb3aa Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Mon, 24 Aug 2015 10:28:39 +0200
Subject: [PATCH] Ensure that the right version of Gtk gets loaded

As suggested by PyGIWarning:

    /usr/lib/python3.4/site-packages/pyfros/controls.py:22: PyGIWarning:
    Gtk was imported without specifying a version first. Use
    gi.require_version('Gtk', '3.0') before import to ensure that the
    right version gets loaded.
          from gi.repository import Gtk

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/pyfros/controls.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/pyfros/controls.py b/src/pyfros/controls.py
index 0593ef7..b64cf8b 100644
--- a/src/pyfros/controls.py
+++ b/src/pyfros/controls.py
@@ -15,6 +15,10 @@
 ## along with this program; if not, write to the Free Software
 ## Foundation, Inc., 51 Franklin Street, Suite 500, Boston, MA  02110-1335  USA
 
+# As suggested by a PyGIWarning
+import gi
+gi.require_version('Gtk', '3.0')
+
 # pylint has troubles importing from gi.repository because
 # it uses introspection
 # pylint: disable=E0611
-- 
2.5.0

